core/src/test/scala/org/apache/spark/benchmark/BenchmarkBase.scala

```diff
@@ -49,7 +49,11 @@ abstract class BenchmarkBase {
       val resultFileName =
         s"${this.getClass.getSimpleName.replace("$", "")}$jdkString$suffix-results.txt"
       val prefix = Benchmarks.currentProjectRoot.map(_ + "/").getOrElse("")
-      val file = new File(s"${prefix}benchmarks/$resultFileName")
+      val dir = new File(s"${prefix}benchmarks/")
+      if (!dir.exists()) {
+        dir.mkdirs()
```

Comment:
Can you add `println` and say the directory is going to be created? e.g.)
```scala
// scalastyle:off println
println(s"Creating ${dir.getAbsolutePath} for benchmark results.")
// scalastyle:on println
```

Reply:
My concern is that the benchmark directory is based on jars paths which are flaky. Might be better to explicitly show.

Reply:
Thanks for the comment :) I added `println` as suggested.